arrowhead-clients/README.md

```diff
@@ -0,0 +1,7 @@
+* The consumer and provider are using [client-library-python](https://github.com/rubenasplund/client-library-python). 
```

Comment:
There was some change mentioned that needed to be applied to this library before it could be used. Please add what need to be changed. Preferrably a readily available patch, or a link to a pre-patched library fork.